### DISCUSSION TITLE
Custom domain HTTPS issues with GoDaddy

### DISCUSSION THREAD

**Discussion by akoskm** [community/community#23814] [Other Feature Feedback, Questions, & Ideas]
<p>Hello,</p>
<p>I tried to follow similar posts on the community forum but nothing really worked so far:</p>
<p><a href="https://github.community/t/how-to-enable-https-support-on-custom-domains/10351/6">https://github.community/t5/GitHub-Pages/How-to-enable-https-support-on-custom-domains/m-p/6923</a></p>
<p>also found some step-by-step instructions in this Gist: <a href="https://gist.github.com/cvan/8630f847f579f90e0c014dc5199c337b" rel="nofollow noopener">https://gist.github.com/cvan/8630f847f579f90e0c014dc5199c337b</a> but I’m not sure what I’m missing.</p>
<p>Here’s my github page: <a href="https://github.com/akoskm/akoskm.github.io" rel="nofollow noopener">https://github.com/akoskm/akoskm.github.io</a> and here is my DNS config from godaddy:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/0e5809a3823e39a54e062994c93b302f7dd25f29.png" data-download-href="/uploads/short-url/22TiS3fgzHlbsFNcfrr0k74AM8V.png?dl=1" title="Screen Shot 2018-08-15 at 12.44.04 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3111627/181079401-6e8fa0ca-c6f2-459a-8a4b-c880d5d105a1.png" alt="Screen Shot 2018-08-15 at 12.44.04 PM.png" data-base62-sha1="22TiS3fgzHlbsFNcfrr0k74AM8V" width="690" height="403" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/0e5809a3823e39a54e062994c93b302f7dd25f29_2_690x403.png, https://aws1.discourse-cdn.com/github/original/1X/0e5809a3823e39a54e062994c93b302f7dd25f29.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/0e5809a3823e39a54e062994c93b302f7dd25f29.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/0e5809a3823e39a54e062994c93b302f7dd25f29_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-08-15 at 12.44.04 PM.png</span><span class="informations">991×579 39.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>the top A record is added when I’m setting up the http-&gt;https forward:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/790aa1a9933b091a244b0b9371c3cfd126f6480a.png" data-download-href="/uploads/short-url/hgMyd67pLP9ytORHd0KvuDfG3Oy.png?dl=1" title="Screen Shot 2018-08-15 at 12.49.55 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3111627/181079402-2277debb-8c3b-4f3e-abff-4366ce2f9fe5.png" alt="Screen Shot 2018-08-15 at 12.49.55 PM.png" data-base62-sha1="hgMyd67pLP9ytORHd0KvuDfG3Oy" width="352" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/790aa1a9933b091a244b0b9371c3cfd126f6480a_2_352x500.png, https://aws1.discourse-cdn.com/github/original/1X/790aa1a9933b091a244b0b9371c3cfd126f6480a.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/790aa1a9933b091a244b0b9371c3cfd126f6480a.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/790aa1a9933b091a244b0b9371c3cfd126f6480a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-08-15 at 12.49.55 PM.png</span><span class="informations">439×623 28.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>even if the box <strong>Update my nameservers and DNS settings to support this change</strong> is unchecked.</p>
<p>Any other GitHub page users facing the same problem?</p>
<p>Thanks,</p>
<p>Akos</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>CNAME records are for subdomains, such as <a href="http://www.example.com" rel="nofollow noopener">www.example.com</a>. A records are for apex domains, such as <a href="http://example.com" rel="nofollow noopener">example.com</a>. It appears from what you’re showing here, that you’ve configured both the A records and CNAME records to point to the apex domain. This may be what is causing your problem. I would recommend simply starting with the CNAME subdomain records, since those are easiest to get working. Once that is working properly, then experiment with the apex domain stuff.</p>
<p>I hope that helps!</p>

**Comment by akoskm** [comment]
<p>If I got you right then this is the configuration I should have for the www address:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/9/9e562fddb0610ff83e0b3c1478d10be04aa6abe0.png" data-download-href="/uploads/short-url/mAI5TtmE8oejtADGrBKdJu9MVgc.png?dl=1" title="Screen Shot 2018-08-19 at 8.34.26 AM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3111627/181079406-b541c616-54be-4815-b8f0-589fb0ed57b7.png" alt="Screen Shot 2018-08-19 at 8.34.26 AM.png" data-base62-sha1="mAI5TtmE8oejtADGrBKdJu9MVgc" width="690" height="281" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/9/9e562fddb0610ff83e0b3c1478d10be04aa6abe0_2_690x281.png, https://aws1.discourse-cdn.com/github/optimized/2X/9/9e562fddb0610ff83e0b3c1478d10be04aa6abe0_2_1035x421.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/9/9e562fddb0610ff83e0b3c1478d10be04aa6abe0.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/9/9e562fddb0610ff83e0b3c1478d10be04aa6abe0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-08-19 at 8.34.26 AM.png</span><span class="informations">1166×475 25.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>does this mean that I should also change the CNAME record in my github repo to <a href="http://www.akoskm.com" rel="nofollow noopener">www.akoskm.com</a>?</p>
<p>Thanks!</p>

**Comment by lee-dohm** [comment]
<blockquote>
<p><a class="mention" href="/akoskm">@akoskm</a> wrote:</p>
</blockquote>
<blockquote>
<p>does this mean that I should also change the CNAME record in my github repo to <a href="http://www.akoskm.com" rel="nofollow noopener">www.akoskm.com</a>?</p>
</blockquote>
<p> Yes, it does.</p>

**Comment by lee-dohm** [comment]
<p>Thanks for joining us, we’re glad you’re here <a class="mention" href="/techboyg5">@techboyg5</a>.</p>
<p>Please don’t “bump” topics by simply commenting on them without adding something useful to the conversation. If the community finds certain discussions useful or important, they’ll naturally bubble to the top. And even when something isn’t discussed often, it can still be found through search.</p>

**Comment by Giovanni945** [comment]
I am a bit confused doesn't putting your domain name in the name field mean that the record will resolve to the domain name twice?

**Comment by TimmieBimmie** [comment]
I don't know if anyone is facing this problem in 2024 but when you put in the custom domain on github you can head over to Cloudflare and change the name-servers from the godaddy name servers to the ones from Cloudflare.
With these you should be able to connect to your website using https. All the warnings should also be gone

Just follow [this](https://www.youtube.com/watch?v=TngeYJmp5pU&ab_channel=KnowledgeBase) tutorial:

If you correctly executed these steps you just need to wait (up to) 24 hours and voilà
